### DISCUSSION TITLE
More precise update download size?

### DISCUSSION THREAD

**Discussion by Qronikarz** [linuxmint/discussions#103] [Ideas]
Hi. So overall I really really like how the update manager includes flatpak updates as you don't have to manually do it from terminal, but they are quite confusing.
![updscreen](https://github.com/linuxmint/discussions/assets/84787215/257700f6-ef68-4ac3-a670-aea12e615f4a)
For example: Size before updating is usually way too big and exaggerated. When you actually install it shows way smaller size in download window and to make it even weirder it can overflow the set size by few times larger number.
Now. From what I remember the terminal version when you run "flatpak update" also wasn't perfectly logical, but could something be done with this issue? Some people need to watch their internet usage to not use more than set limit.

Also, while I'm at similar topic. Sometimes there are normal package updates that show as 1-5 MB in update manager but when you install them they suddenly decide it's something like 100-200MB. If remember correctly it's either linux kernel or linux-firmware package. What is the issue there?

**Comment by ElmNumaeStNo37** [comment]
To my knowledge this is caused by flatpak using delta updates and it calculations is based on the largest possible size of the application update but normally passes the size of the commit done for the said update. Unfortunately, i don't think Mint dev team has control on this, perhaps you can also request this feature in flatpak's github

**Comment by Qronikarz** [comment]
Got it. Will post the flatpak part in Flatpak GitHub.

That only leaves linux kernel and firmware packages at Mint control then.

update: Posted here https://github.com/flatpak/flatpak/issues/2627#issuecomment-1677372822
